01-coordinated-multipoint.ipynb

```diff
@@ -631,7 +631,7 @@
    "metadata": {},
    "source": [
     "Processing time in Wi-Fi/cellular networks is crucial: base stations or C-RANs must \n",
-    "decode transmissions in tens of microseconds. \n",
+    "decode transmissions in tens of milliseconds. \n",
```

Comment:
I would say 'at most tens of milliseconds'
We know of requirements that are shorter, but not longer: '10 ms delay for WCDMA and 3 ms for LTE'

Reply:
Done